integration_test/common/common.go

```diff
@@ -37,13 +40,15 @@ type ExpectedMetric struct {
 	MonitoredResource string `yaml:"monitored_resource" validate:"required,oneof=gce_instance"`
 	// Mapping of expected label keys to value patterns.
 	// Patterns are RE2 regular expressions.
-	Labels map[string]string `yaml:"labels" validate:"required"`
+	Labels map[string]string `yaml:"labels,omitempty" validate:"required"`
 	// If Optional is true, the test for this metric will be skipped.
-	Optional bool `yaml:"optional" validate:"excluded_with=Representative"`
+	Optional bool `yaml:"optional,omitempty" validate:"excluded_with=Representative"`
 	// Exactly one metric in each expected_metrics.yaml must
 	// have Representative set to true. This metric can be used
 	// to test that the integration is enabled.
-	Representative bool `yaml:"representative" validate:"excluded_with=Optional"`
+	Representative bool `yaml:"representative,omitempty" validate:"excluded_with=Optional,excluded_with=Platform"`
+	// Exclusive metric to a particular kind of platform
```

Comment:
[Optional] All other such comments have a trailing '.'…

Reply:
Changed in: [cf8f612](https://github.com/GoogleCloudPlatform/ops-agent/pull/719/commits/cf8f61234082bc521e24290d1f64d70b9a784454)